docs/en/enterprise-edition/content-collections/administration/create-and-manage-saved-views.adoc

```diff
@@ -0,0 +1,43 @@
+== Create and Manage Saved Views
+
+Prisma Cloud offers the ability to create *Saved Views* to prioritize and organize your data into categories that are meanigful to your organization. Saved views are available on the Governance, Alerts, Compliance and Dashboards pages. For instance on the Alerts page Prisma Cloud provides a set of default views — Overview, Misconfigurations, Incidents, Malware, and Highest Priority, but you can also create a custom view and share it with others in your organization or tenant. Review the Alerts page workflows below to create, manage, and share your Saved Views.
+
+
+[NOTE]
+====
+Default (*System*) views are initially the opinionated suggestion of what a view should be based upon the problem being solved. Views such as *High-Priority*, *Attack Paths*, and *Incidents* etc, include preset filters that display the most relevant alerts for each category. If you make changes to a Default view, the changes are persisted each time you return to that view. Default views do not reset to the initial System view and cannot be deleted or reordered. *Overview*, which is the first view, also cannot be hidden.
```

Comment:
Suggest a minor re-write:
If you make changes to a default view, those changes are saved and will persist the next time you view it.